File: pset6/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(void);

int main(void)
{
    int t = get_positive_int();


    for (int i = 1; i <= t; i++)
    {
        for (int j = i ; j < t; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}


// Prompt user for positive integer
int get_positive_int(void)
{
    int n;
    do
    {
        n = get_int("Positive Integer, between 1 to 8 inclusive: ");
    }
    while (n < 1 || n > 8);
    return n;
}   

File: pset6/mario/less/mario.py
from cs50 import get_int


def main():
    t = get_positive_int()

    for i in range(1, t+1, 1):
        for j in range(i, t, 1):
            print(" ", end="")
        for k in range(i):
            print("#", end="")
        print()


def get_positive_int():
    while True:
        n = get_int("Positive Integer, between 1 to 8 inclusive: ")

        if n > 0 and n < 9:
            break
    return n


main()